packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Customize the sign-up link in the Widget
+---
+
+If you left the <GuideLink link="../configure-self-service-registration-policy">**Add to Sign-In widget**</GuideLink> check box clear when configuring the registration policy, then you need to configure a link using JavaScript in the **Customize Sign-In Page** HTML editor.
+
+> **Note:** To enable the Custom Sign-In Page HTML editor, you must have a [custom URL domain](/docs/guides/custom-url-domain/overview/) configured.
+
+1. In the Admin Console, go to **Settings** and then **Customization**.
```

Comment:
Another Developer vs Classic UI issue
(i'm not sure the best way to deal with these)

Reply:
@briandemers-okta - They can't use the Developer Console for these steps. They have to use the Admin Console (Classic UI). I will put a note at the very top about performing these steps in the Admin Console so that users aren't confused.